src/build/app/settings.rs

```diff
@@ -666,7 +666,7 @@ pub enum AppSettings {
     ///
     /// # Panics
     ///
-    /// ```rust,should_panic
+    /// ```rust,no_run
```

Comment:
I'd rather leave it as `should_panic`. Tests are run only on debug, aren't they?

Reply:
I did this because I added release profile tests too.